.github/workflows/e2e.yml

```diff
@@ -6,7 +6,7 @@ name: E2E tests
 permissions: read-all
 
 env:
-  DEBUG: 'bidi:server:info,bidi:mapper:debug:*'
+  DEBUG: 'bidi:server:*,bidi:mapper:debug:*'
```

Comment:
Decided to go with this rather then update `DEBUG` when `VERBOSE` so they don't depend on each. Else it becomes problematic with mix and matching.